superflore/generators/bitbake/ros_meta.py

```diff
@@ -74,17 +83,28 @@ def get_file_revision_logs(self, *file_path):
     def get_change_summary(self, distro):
         self.repo.git.add('meta-ros{0}-{1}/generated-recipes'.format(
             yoctoRecipe._get_ros_version(distro), distro))
+        self.repo.git.add('meta-ros{0}-{1}/conf/ros-distro/include/{1}/'
```

Comment:
Why was this moved here? It seems confusing that the function `commit_changes()` doesn't commit all of the changes.

Reply:
I've moved this to separate PR to avoid confusion.